### PR TITLE
feat: [PL-26581]: Showing Platform service versions in Account overview page

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
#### Screenshots

Screenshot 2022-11-04 at 3 52 30 PM

Screenshot 2022-11-04 at 3 43 08 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/33-auth-settings/pages/AccountOverview/AccountOverview.tsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ const AccountOverview: React.FC = () => {
       <Page.Body>
         <AccountDetails />
         {(createdFromNG || NG_LICENSES_ENABLED) && <SubscribedModules />}
+        {<ServiceVersions />}
```
Comment (abhinavrastogi-harness) [inline src/modules/33-auth-settings/pages/AccountOverview/AccountOverview.tsx:34]: move this below AccountSettings

File: src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  const [loading, setLoading] = useState<boolean>()
+  const { getString } = useStrings()
+
+  const fetchServices = () => {
+    const servicesLength = Object.keys(data).length
+    if (servicesLength <= 0) {
+      const promiseArr = versionAPIs.map(row => fetch(row.url.startsWith('http') ? row.url : BASE_URL + row.url))
+
+      setLoading(true)
+      Promise.all(promiseArr).then(async responses => {
```
Comment (abhinavrastogi-harness) [inline src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx:105]: error handling for this?

File: src/modules/33-auth-settings/pages/AccountOverview/AccountOverview.tsx (hunk 1 above)
Comment (avinashmadhwani02) [reply src/modules/33-auth-settings/pages/AccountOverview/AccountOverview.tsx]: Done

**Review by devinder-harness — DISMISSED** [review]

File: src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx (hunk 2 above)
Comment (avinashmadhwani02) [reply src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx]: Added catch and showing error if any of the APIs fails.

Comment (avinashmadhwani02) [reply src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx]: Ideally, if one API is failing we should show versions of other APIs.
Let me check if I can make that change, We can merge this. I will do it in a separate PR

File: src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+import css from '../AccountOverview.module.scss'
+
+const versionAPIs = [
+  {
+    label: 'Access Control',
+    url: 'authz/api/version',
+    id: 'access_control'
+  },
+  {
+    url: 'https://localhost:8181/auth/version.json',
```
Comment (mayankVermaHarness) [inline src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx:22]: Is this localhost URL intended?

File: src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx (hunk 2 above)
Comment (mayankVermaHarness) [reply src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx]: +1

**Review by mayankVermaHarness — DISMISSED** [review]

**Review by mayankVermaHarness — APPROVED** [review]

**Review by mayankVermaHarness — APPROVED** [review]

**Review by devinder-harness — APPROVED** [review]

**Pull request merged**